src/ol/events/condition.js

```diff
@@ -239,7 +239,7 @@ export const shiftKeyOnly = function (mapBrowserEvent) {
 
 /**
  * Return `true` if the target element is not editable, i.e. not a `<input>`-,
- * `<select>`- or `<textarea>`-element, `false` otherwise.
+ * `<select>`-, `<textarea>`-element or `contentEditable` -attribut at true, `false` otherwise.
```

Comment:
I would change this to be “not an `input`, `select`, or `textarea` element and no `contenteditable` attribute.”

Reply:
@tschaub thanks ==> done